src/TileChartMap.less

```diff
@@ -146,10 +146,6 @@ x/y positioning on the SVG elements.
         .legend-box {
             stroke: @gray;
         }
-
-        .legend-text {
-            transform: translate(3px, -3px);
-        }
```

Comment:
In the screenshots, the tile text looks shifted.  Do we need to restore this?

Reply:
this is only for the legend bin values, and it was moved in the legend code here https://github.com/cfpb/ccdb5-ui/pull/191/files#diff-a7f3f10a59c833f0a7e2a676fcff76f4R288

For some reason IE11 wasn't respecting this css rule